source/administration/release-definitions.rst

```diff
@@ -0,0 +1,21 @@
+Release Definitions
+========================
+
+Mattermost adopts a monthly tick-tock release cycle, with a new version shipping on the 16th of each month in `binary form <http://docs.mattermost.com/administration/upgrade.html>`_.
+
+Tick-tock refers to even-numbered releases (e.g., v5.6) containing new features, and odd-numbered releases (e.g., v5.7) containing only bug fixes and performance improvements.
```

Comment:
Should we replace this with "5.24" and "5.25" as that's how we name our versions (0/5 on which version numbers we use in the example).